<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettingsNotification</class>
 <widget class="QWidget" name="UISettingsNotification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Enable notifications</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="5">
           <widget class="QCheckBox" name="checkBox_NICKSAY">
            <property name="text">
             <string>Your nick is mentioned in main chat</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QCheckBox" name="checkBox_PM">
            <property name="text">
             <string>Private message</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="5">
           <widget class="QCheckBox" name="checkBox_TRDONE">
            <property name="text">
             <string>Transfer done</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="5">
           <widget class="QCheckBox" name="checkBox_ANY">
            <property name="text">
             <string>Allow other types of notifications</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Notification module</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Default (Qt)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>D-Bus</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Sound</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_SND">
         <property name="title">
          <string>Enable sounds</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_SNDNICKSAY"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_TESTNICKSAY">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolButton_BRWNICK">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineEdit_SNDPM"/>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_TESTPM">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="toolButton_BRWPM">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="lineEdit_SNDTRDONE"/>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_TESTTR">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QToolButton" name="toolButton_BRWTR">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Your nick is mentioned in main chat</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Private message</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Transfer done</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
